aplt/logobserver.py

```diff
@@ -93,9 +95,19 @@ def json_format(self, event):
         if 'reason' in event:
             event['reason'] = repr(event['reason'])
         event['message'] = ev
-        event['log_level'] = event['log_level'].name
+        try:
+            event['log_level'] = event['log_level'].name
```

Comment:
I see what was happening here, the AP_Logger was probably registered as an observer multiple times during tests. json_format modifies the event dict in place here so this was working around json_format being called a second time on the same, now modified event dict.

Best thing to avoid all of that is to just create a new dict for the JSON dumps, leaving the event dict unmodified. autopush PushLogger actually does this